### DISCUSSION TITLE
xiaomi aircondition for python

### DISCUSSION THREAD

**Discussion by sufeicn** [rytilahti/python-miio#1778] [Q&A]
How to define object. How to check the status。 thx

**Comment by rytilahti** [comment]
Using the current git master branch, you can do the following:
```
from miio import DeviceFactory

dev = DeviceFactory.create("<addr>", "<token>")
print(dev.status())
```

The newest released version does not have devicefactory, so you'll need to figure out whichever class supports the device and use that directly. To find out the class, you can use  `miiocli device --ip <addr> --token <token> info` on the current git master, or just grep through the code base searching for the model the info gives you.
